src/Catty/PlayerEngine/Scene/CBScene.swift

```diff
@@ -60,6 +62,18 @@ final class CBScene: SKScene {
     // MARK: - Deinitializer
     @objc deinit { logger.info("Dealloc Scene") }
 
+    override func update(_ currentTime: TimeInterval) {
+
+        if frameCounter == PlayerConfig.NumberOfFramesPerSpriteNodeUpdate {
```

Comment:
Just to be on the safe side (in case one execution is missed or similar), I would rather go for `>=`.